tests/fixtures/mocks.py

```
@@ -325,17 +381,47 @@ def mock_async_setup(entry_id):
 
-            # Mock device methods
-            mock_device = MagicMock()
-            mock_device.get_sensors_by_platform.return_value = [MagicMock()]
-            mock_device.get_name.return_value = "device_name"
+            # Try to use patched ImouDevice if available, otherwise create a mock
+            try:
+                from imouapi.device import ImouDevice
+
+                # This will use the patched ImouDevice if the test patched it
+                mock_device = ImouDevice(None, None)
+            except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)